cupy/cuda/stream.pyx

```diff
@@ -303,11 +313,14 @@ class Stream(BaseStream):
 
     """
 
-    def __init__(self, null=False, non_blocking=False):
+    def __init__(self, null=False, non_blocking=False, ptds=False):
```

Comment:
Let's add ptds to the docstring of `Stream`

Reply:
Done in the latest commits, please note I did a few changes to clarify the order that arguments respect.